backend/modules/auth/router.py

```
@@ -1,33 +1,656 @@
+                "google_id": google_id,
+                "email": oauth_user_data.email,
+                "name": oauth_user_data.full_name,
+                "picture": oauth_user_data.avatar_url
+            }
+        }
+        
+        return create_role_selection_redirect(role_selection_data)
+    
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)